Does not build with highs crate on minimum version

I was getting build errors when I used good_lp in my project, but it built fine when using the good_lp crate on its own.

Steps to reproduce:
```diff
-highs = { version = "2.0.0", optional = true }
+highs = { version = "=2.0.0", optional = true }
```
Error:
```bash
error[E0599]: no method named `try_set_option` found for struct `highs::Model` in the current scope
   --> src/solvers/highs.rs:292:54
    |
292 |                 HighsOptionValue::String(v) => model.try_set_option(k.as_str(), v.as_str()),
    |                                                      ^^^^^^^^^^^^^^
    |
help: there is a method `set_option` with a similar name
    |
292 -                 HighsOptionValue::String(v) => model.try_set_option(k.as_str(), v.as_str()),
292 +                 HighsOptionValue::String(v) => model.set_option(k.as_str(), v.as_str()),
    |

error[E0599]: no method named `try_set_option` found for struct `highs::Model` in the current scope
   --> src/solvers/highs.rs:293:53
    |
293 |                 HighsOptionValue::Float(v) => model.try_set_option(k.as_str(), v),
    |                                                     ^^^^^^^^^^^^^^
    |
help: there is a method `set_option` with a similar name
    |
293 -                 HighsOptionValue::Float(v) => model.try_set_option(k.as_str(), v),
293 +                 HighsOptionValue::Float(v) => model.set_option(k.as_str(), v),
    |

error[E0599]: no method named `try_set_option` found for struct `highs::Model` in the current scope
   --> src/solvers/highs.rs:294:52
    |
294 |                 HighsOptionValue::Bool(v) => model.try_set_option(k.as_str(), v),
    |                                                    ^^^^^^^^^^^^^^
    |
help: there is a method `set_option` with a similar name
    |
294 -                 HighsOptionValue::Bool(v) => model.try_set_option(k.as_str(), v),
294 +                 HighsOptionValue::Bool(v) => model.set_option(k.as_str(), v),
    |

error[E0599]: no method named `try_set_option` found for struct `highs::Model` in the current scope
   --> src/solvers/highs.rs:295:51
    |
295 |                 HighsOptionValue::Int(v) => model.try_set_option(k.as_str(), v),
    |                                                   ^^^^^^^^^^^^^^
    |
help: there is a method `set_option` with a similar name
    |
295 -                 HighsOptionValue::Int(v) => model.try_set_option(k.as_str(), v),
295 +                 HighsOptionValue::Int(v) => model.set_option(k.as_str(), v),
    |

For more information about this error, try `rustc --explain E0599`.
error: could not compile `good_lp` (lib) due to 4 previous errors
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -40,7 +40,7 @@ minilp = [
 coin_cbc = { version = "0.1", optional = true, default-features = false }
 microlp = { version = "0.4.0", optional = true }
 lpsolve = { version = "1.0.1", optional = true }
-highs = { version = "2.0.0", optional = true }
+highs = { version = "2.2.0", optional = true }
 russcip = { version = "0.9.1", optional = true }
 lp-solvers = { version = "1.0.0", features = ["cplex"], optional = true }
 cplex-rs = { version = "0.1.9", optional = true }

